### DISCUSSION TITLE
疑问

### DISCUSSION THREAD

**Discussion by imokokok** [smartcontractkit/Web3_tutorial_Chinese#22] [Q&A]
第四节课五十分钟为什么我使用npx hardhat一直出现这样的情况
![屏幕截图 2024-11-30 161258](https://github.com/user-attachments/assets/caa89a36-06cd-456d-8fcb-a3e9c80f70e1)

**Comment by QingyangKong** [comment]
先检查是否已经成功安装了 Hardhat
然后检查版本是否和 repo package.json 的版本一致
试试 npx hardhat init 是否新建一个项目
